orthoinference/src/main/java/org/reactome/orthoinference/OrthologousEntityGenerator.java

```diff
@@ -189,28 +194,31 @@ private static GKInstance createInfComplexPolymer(GKInstance complexInst, boolea
 			{
 				if ((complexTotalProteinCounts > 0 && complexInferrableProteinCounts == 0) || percent < 75)
 				{
-					logger.info("\tComplex/Polymer protein count is below threshold -- terminating inference");
+					logger.info("Complex/Polymer protein count is below 75% threshold (" + percent + "%) -- terminating inference");
 					return nullInst;
 				}
 			}
-			
+			logger.info("Complex protein counts. Total: " + complexTotalProteinCounts + "  Inferrable: " + complexInferrableProteinCounts);
 			GKInstance infComplexInst = InstanceUtilities.createNewInferredGKInstance(complexInst);
 			infComplexInst.addAttributeValue(summation, complexSummationInst);
 			infComplexInst.addAttributeValue(name, complexInst.getAttributeValue(name));
-			ArrayList<GKInstance> infComponentInstances = new ArrayList<GKInstance>();
+			ArrayList<GKInstance> infComponentInstances = new ArrayList<>();
 			// Inference handling is different depending on if it is a Complex or a Polymer. Complexes will infer all 'components' while Polymers will infer all 'repeatedUnits'.
+			// TODO: Log the ratio of inferred complex/polyer from human?
 			if (complexInst.getSchemClass().isa(Complex))
 			{
-				logger.info("\tInferring Complex components...");
-				for (Object componentInst : complexInst.getAttributeValuesList(hasComponent))
+				Collection<GKInstance> componentInstances = complexInst.getAttributeValuesList(hasComponent);
+				logger.info("Complex components: " + componentInstances);
+				for (Object componentInst : componentInstances)
```

Comment:
This declaration can be a GKInstance instead of an Object so there's no need for a cast.